### DISCUSSION TITLE
Error 'SequentialDataset' object has no attribute 'item_id_list_field'

### DISCUSSION THREAD

**Discussion by VladHardAndSoftWare** [RUCAIBox/RecBole#1623] [Q&A]
When I point `benchmark_filename: ['part1', 'part2', 'part3']`  in config.yaml, I get this error message:` AttributeError: 'SequentialDataset' object has no attribute 'item_id_list_field' `
![image](https://user-images.githubusercontent.com/42782084/211029124-59a92600-d2ae-48c1-bb94-40b3f0b1e6ea.png)

**Comment by AoiDragon** [comment]
Hello @VladHardAndSoftWare, you can find explanation in #1624 .
